impl/src/main/java/io/astefanutti/metrics/cdi/MetricsConfigurationEvent.java

```diff
@@ -36,10 +39,21 @@ public MetricsConfiguration useAbsoluteName(boolean useAbsoluteName) {
         return this;
     }
 
+    @Override
+    public MetricsConfiguration useReservoirBuilder(ReservoirBuidler builder) {
+        throwsIfUnmodifiable();
+        this.reservoirBuilder = builder;
+        return this;
+    }
+
     Set<MetricsParameter> getParameters() {
         return Collections.unmodifiableSet(configuration);
     }
 
+    ReservoirBuidler getReservoirBuilder() {
```

Comment:
I created an EnumSet so that we could add other configuration parameters. I wonder whether we change it to EnumMap and add the `ReservoirBuidler` configuration to it. The idea being that it avoids having to much getters.